CONTRIBUTING.md

```diff
@@ -1,49 +1,53 @@
-## Contributing
```

Comment:
The headline levels are intentionally down one level because this is _part of_ a book, and the headlines participate in the whole.

If you take a look at [the current version on gitbooks](https://poshcode.gitbooks.io/powershell-practice-and-style/content/CONTRIBUTING.html) I think it's correct --see the side-bar TOC, in particular. I believe that changing the headline level would mess up the book. I guess I could merge this into a branch and test it.